src/server/serverless/bulk-qrcode-generation/index.js

```diff
@@ -0,0 +1,42 @@
+const { ImageFormat, createQRCodesBuffer } = require('./qrCode')
+const { createCsv } = require('./createCsv')
+const { uploadToS3 } = require('./s3')
+const { sendSQSMessage } = require('./sqs')
+
+async function handler(event) {
+  const { body } = event.Records[0]
```

Comment:
Nit, can add a short inline comment that this currently assumes Records batch size of 1?